namespaces/live-1.cloud-platform.service.justice.gov.uk/track-a-query-staging/resources/s3.tf

```diff
@@ -61,4 +61,20 @@ resource "kubernetes_secret" "track_a_query_s3" {
     bucket_arn        = "${module.track_a_query_s3.bucket_arn}"
     bucket_name       = "${module.track_a_query_s3.bucket_name}"
   }
+
+  cors_rule = [
```

Comment:
Sorry I missed this. 
cors_rule should not be in the kubernetes_secret, but in the module block